pkg/controller/container-runtime-config/helpers.go

```diff
@@ -1076,3 +1110,75 @@ func addScopeMirrorsSigstoreRegistriesdConfig(registriesDockerConfig map[string]
 	}
 	return nil
 }
+
+func updateNamespacedPolicyJSONs(clusterOvrridePolicyJSON []byte, internalBlocked, internalAllowed []string, scopeNamespacePolicies map[string]map[string]signature.PolicyRequirements) (map[string][]byte, error) {
+	// updatePolicyJSON should make sure the default cluster policy or cluster overiide policy is not empty
```

Comment:
nit: "overiide" -> "override"

Reply:
explain in the updateNamespacedPolicyJSONs method comment